Makefile.am

```diff
@@ -5,14 +5,14 @@ SUBDIRS = lib src man
 EXTRA_DIST = m4/gnulib-cache.m4
 
 install-exec-local:
-	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli gfcat
-	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli gfcp
-	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli gfls
-	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli gfmkdir
-	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli gfmv
-	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli gfrm
-	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli gfstat
-	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli gftail
+	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli $(DESTDIR)$(bindir)/gfcat
+	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli $(DESTDIR)$(bindir)/gfcp
+	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli $(DESTDIR)$(bindir)/gfls
+	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli $(DESTDIR)$(bindir)/gfmkdir
+	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli $(DESTDIR)$(bindir)/gfmv
+	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli $(DESTDIR)$(bindir)/gfrm
+	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli $(DESTDIR)$(bindir)/gfstat
+	$(LN_S) -f $(DESTDIR)$(bindir)/gfcli $(DESTDIR)$(bindir)/gftail
```

Comment:
Sorry guys. My understanding was wrong. Please read through the section titled 'Test-Driving a Relocatable Package' from the following link:

http://www.rpm.org/max-rpm/s1-rpm-reloc-building-relocatable.html

> RPM blindly packages the symlink, complete with its non-relocatable string. This is why we mentioned absolute symlinks as a prime example of non-relocatable software. Fortunately, this problem isn't that difficult to fix. All we need to do is change the line in the makefile that creates the symlink.

So everything is clear and @nixpanic is 100% correct here. Therefore I am updating the PR with relative symlink.